Run Prettier on JS code fences, part 1

OnkarRuikar opened the pull request:
The PR focuses only on JS code fences.

Idea is to gradually prettify all the JS code fences before the full automation.

Note: pulling only cosmetic/simple changes.

bsmth commented:
Thanks @OnkarRuikar, taking a look 👀

bsmth reviewed (approved):
Looks good to me, thanks 👍🏻

Josh-Cena commented on files/en-us/mozilla/add-ons/webextensions/api/action/openpopup/index.md:
```diff
@@ -43,8 +43,8 @@ Open the popup when the user selects a context menu item:
   id: "open-popup",
-  title: "open popup",
-  contexts: ["all"]
+  title: 'open popup',
```
Why is this changing from double to single quote?

OnkarRuikar replied:
Ohh. Changes from other PR, related to frontmatter titles, overlapped.
I've rerun the tool and it has resolved it.

OnkarRuikar committed:
Run Prettier on JS code fences, part 1

The PR focuses only on JS code fences.

Idea is to gradually prettify all the JS code fences before the full automation.
